wormhole-connect/src/config/index.ts

```diff
@@ -31,6 +31,7 @@ const getEnv = () => {
 export const ENV = getEnv();
 export const isMainnet = ENV === 'MAINNET';
 export const sdkConfig = WormholeContext.getConfig(ENV);
+export const pageHeader = config.pageHeader || 'Bridge';
```

Comment:
🤔 should we allow integrators to remove the page header?

Reply:
e.g. 
```
export const pageHeader = config.pageHeader === undefined ? 'Bridge' :  config.pageHeader;
```

Reply:
JK - now the default is the empty string